### DISCUSSION TITLE
Can the single function app handle multiple key vault

### DISCUSSION THREAD

**Discussion by ash06dec** [polymind-inc/acmebot#559] [Q&A] [answered]
Is there any option to have multiple key vault in the single function app? Considering changing the variable "Acmebot:VaultBaseUrl". Please confirm.

**Comment by shibayan** [comment, marked as answer]
Support for multiple Key Vaults is not planned.

Since Acmebot is very low cost, we recommend deploying it on a one-to-one basis with Key Vaults.
